docs/changelog-fragments.d/194.bugfix.md

```diff
@@ -0,0 +1,3 @@
+Ensure ANSI escapes are removed from stdout/stderr
```

Comment:
It's not a Git commit message, use the past tense:
```suggestion
Started removing ANSI escapes from stdout/stderr
```

Reply:
Also, this seems to be missing the specifics of when the stripping-off happens. Is it for all of the subprocesses? Only Ansible ones? Always?

Reply:
Alternatively, this could be phrased as
```suggestion
From this point forward the ANSI escape codes are removed from the
standard output and error streams of all directly invoked subprocesses
```
(IIUC)

Reply:
That is a library, not using stderr and stdout would be worse from the usability point of view.

Reply:
I don't think so. If a person doesn't know that stderr == standard error, they probably don't even care about this change. Still, the guidance is clear — avoid the use of short versions for things, even stuff like eg/ie/etc.